### PR TITLE
Added Styles File

### PR DESCRIPTION
**Pull request by Adeoye-J**
Kindly look through it.

### CODE REVIEW

**Review by Adeoye-J — COMMENTED** [review]
Looks good to me

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <body>
     <h1>Lets have fun learning how to collaborate</h1>
     <h2>Lets have fun learning how to collaborate</h2>
+    <p>Changes made</p>
```
[the remark this answers is not in the document]
Comment (TechWithJerry) [reply index.html:11]: You resolved nothing

**Review by TechWithJerry — CHANGES_REQUESTED** [review]
Make sure to complete all requested action before resolving.

**Review by TechWithJerry — APPROVED** [review]
Changes made

**Pull request merged**